src/components/Image/index.js

```diff
@@ -56,9 +57,10 @@ class Image extends React.Component {
     render() {
         // eslint-disable-next-line
         const { source, onLoad, ...rest } = this.props;
```

Comment:
This is not part of your changes, but can we cleanup this by using `omit` instead? `source` and `onLoad` has not been used.

Reply:
Done